src/main/java/com/google/sps/servlets/NewDrugServlet.java

```diff
@@ -43,18 +43,31 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Sanitize user input to remove HTML tags and JavaScript.
     String name = getCleanParameter(request, "name");
+    String classification = getCleanParameter(request, "classification");
     List<StringValue> aliases = splitString(getCleanParameter(request, "aliases"));
+    String safetyDescription = getCleanParameter(request, "safetyDescription");
+    List<StringValue> drugWarningSigns = splitString(getCleanParameter(request, "drugWarningSigns"));
     String description = getCleanParameter(request, "description");
     List<StringValue> effects = splitString(getCleanParameter(request,"effects"));
+    List<StringValue> overdoseSigns = splitString(getCleanParameter(request, "overdoseSigns"));
+    List<StringValue> legalLocations = splitString(getCleanParameter(request, "legalLocations"));
+    List<StringValue> decriminalizedLocations = splitString(getCleanParameter(request, "decriminalizedLocations"));
+
 
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
     KeyFactory keyFactory = datastore.newKeyFactory().setKind("Drug");
     FullEntity drugEntity = 
         Entity.newBuilder(keyFactory.newKey())
             .set("Name", name)
+            .set("Classification", classification)
             .set("Aliases", aliases)
+            .set("Safety Description", safetyDescription)
```

Comment:
I think this works, but I would avoid spaces in property names. I would probably just skip the spaces and use SafetyDescription. The same for the remaining properties.

Reply:
I got rid of the spacing. Do I make a new PR for these changes?

Reply:
No, you can add the commit to this branch and push to update this PR.